.circleci/config.yml

```diff
@@ -152,7 +152,22 @@ jobs:
       - *init_bazel
       - *hide_node_and_yarn_local_binaries
 
-      - run: bazel test ... --test_tag_filters=-e2e,-examples
+      - run: bazel test --test_tag_filters=-e2e,-examples ...
+
+      - store_artifacts:
+          path: dist/bin/release.tar.gz
+          destination: release.tar.gz
+
+  test_debug:
+    <<: *job_defaults
+    resource_class: xlarge
+    steps:
+      - *attach_workspace
+      - *init_environment
+      - *init_bazel
+      - *hide_node_and_yarn_local_binaries
+
+      - run: bazel test --define=DEBUG=1 --define=VERBOSE_LOGS=1 --test_tag_filters=-e2e,-examples ... //packages/terser/test/debug:test_define_DEBUG
```

Comment:
so much CI load!
aren't we at the point of only adding things to buildkite now?

Reply:
yes. I think we can probably start taking CircleCI down. only a few assertions left there that are not covered on buildkite.

Reply:
I'll remove this job and rest can be removed separately